components/Navbar.tsx

```diff
@@ -41,32 +42,42 @@ export function Navbar() {
 		<header className='p-4 flex-[0.25] fixed top-0 z-50 w-screen'>
 			<nav
 				className={`${
-					navbar ? 'max-h-screen' : 'max-h-16'
+					navbar ? 'max-h-screen' : 'max-h-20'
 				} bg-dark-600/50 p-4 rounded-md max-w-5xl mx-auto backdrop-blur-lg transition-all duration-500 ease-in-out overflow-hidden`}
 			>
 				<div className='flex flex-row justify-between items-center'>
-					<div>Logo</div>
-					<div className='sm:hidden text-light-500 text-2xl h-6'>
+					<div className='flex flex-row items-center gap-x-2'>
+						<Link href='/'>
+							<Image
```

Comment:
Not sure about using this logo, I think we should try to get a non-stacked version similar to [this](http://citycyclery.ca/wp-content/uploads/2019/02/cropped-city-cylery-logo-blue.png).

Reply:
I agree!